tmail-backend/guice/encrypted-mailbox/src/test/java/com/linagora/tmail/module/LinagoraTestJMAPServerModule.java

```diff
@@ -0,0 +1,48 @@
+package com.linagora.tmail.module;
+
+import java.util.Optional;
+
+import javax.inject.Singleton;
+
+import org.apache.james.modules.TestJMAPServerModule;
+
+import com.google.inject.Provides;
+import com.linagora.tmail.james.jmap.jwt.JwtPrivateKeyConfiguration;
+import com.linagora.tmail.james.jmap.jwt.JwtPrivateKeyProvider;
+
+import scala.jdk.javaapi.OptionConverters;
+
+public class LinagoraTestJMAPServerModule extends TestJMAPServerModule {
```

Comment:
Why putting this in the encrypted-mailbox guice module? It has nothing to do with the encrypted mailbox right?

Reply:
I thought `encrypted-mailbox` is regarding about all tmail-mailbox.
I will create a new submodule `guice/jmap` for it